mergify_engine/context.py

```diff
@@ -788,6 +790,69 @@ async def retrieve_unverified_commits(self) -> typing.List[str]:
             if not commit["commit_verification_verified"]
         ]
 
+    async def retrieve_resolved_review_threads(
+        self, resolved: typing.Optional[bool] = True
+    ) -> typing.List[str]:
+        query = """
+            repository(owner: "{owner}", name: "{name}") {{
+                pullRequest(number: {number}) {{
+                    reviewThreads(first: 100{after}) {{
+                    edges {{
+                        node {{
+                        isResolved
+                        comments(first: 1) {{
+                            edges {{
+                                node {{
+                                    body
+                                }}
+                            }}
+                        }}
+                        }}
+                    }}
+                    }}
+                }}
+            }}
+        """
+        responses = typing.cast(
+            typing.AsyncIterable[
+                typing.Dict[str, github_graphql_types.GraphqlRepository]
+            ],
+            multi.multi_query(
+                query,
+                iterable=(
+                    {
+                        "owner": self.repository.repo["owner"]["login"],
+                        "name": self.repository.repo["name"],
+                        "number": self.pull["number"],
+                    },
+                ),
+                send_fn=self.client.graphql_post,
+            ),
+        )
+        resolved_review_threads = []
+        query_number = 0
+        current_query = "Q" + str(query_number)
```

Comment:
These variables are not needed. You already have the `response[current_query]` value on each loop, thank to `.items()`

https://docs.python.org/3/tutorial/datastructures.html#looping-techniques

```suggestion
```